lib/opentok/token_generator.rb

```diff
@@ -81,6 +81,15 @@ def generate_token
         end
         data_params[:connection_data] = data
       end
+
+      if opts[:initial_layout_classes]
+        if opts[:initial_layout_classes].is_a?(Array)
+          data_params[:initial_layout_class_list] = opts[:initial_layout_classes].join(' ')
+        else
```

Comment:
can we do `elsif opts[:initial_layout_classes].is_a?(String)` to further filter out other data type?